using-timescaledb/distributed-hypertables.md

```diff
@@ -103,6 +103,90 @@ the data node's data is outside the retention window,
 [`drop_chunks`][drop_chunks] can be used to delete the data on the data
 node.
 
+### Replication Factor for Distributed Hypertables [](replication-factor)
+
+A hypertable can have a *replication factor* that tells how many
+replicas of chunks that should be created. The default replication
+factor is 1, which means that each row of data is written to a single
+chunk. If you set the replication factor of a hypertable to a higher
+value, chunks will be replicated and each row of data will be written
+to that many chunks.
+
+With a replication factor of one, data is written to the data nodes
+using a one-phase commit, which basically means that every transaction
+on the access node opens a similar transaction on the data
+node. Committing the transaction involves sending a commit message to
+the data node and no additional coordination.
+
+With a higher replication factor, data will be written to the chunks
+using two-phase commit. This protocol is more complicated, but it
```

Comment:
Again, the replication_factor has no connection to transactional semantics and commit protocol used.

Reply:
Removed paragraphs.